src/main/cpp/Climber.cpp

```diff
@@ -4,13 +4,15 @@ Climber::Climber(ClimberConfig config) : _config(config) {}
 
 void Climber::OnUpdate(units::second_t dt) {
   switch (_state) {
-    case ClimberState::kIdle: 
-    break;
-    case ClimberState::kClimb:
-    break;
-    case ClimberState::kHang:
+    case ClimberState::kIdle:
+    {
+      _config.climberGearbox.motorController->Set(0);
+    }
     break;
     case ClimberState::kRaw:
+    {
+      _config.climberGearbox.motorController->Set(_rawVoltage.value());
```

Comment:
why are you using Set and not SetVoltage?

Reply:
That is because the motor controller for frc uses Set() intead of setVoltage()

Reply:
This isn't true MotorController supports SetVoltage as well